client/src/modules/lead/EditLead/EditLeadContent/EditLeadTabs/AddActivity/LinkedTo/LeadAutocompleteWrapper/LeadAutocompleteWrapper.tsx

```diff
@@ -0,0 +1,133 @@
+}
+
+interface State {
+  isDropdownOpen: boolean;
+  selectedLead?: any;
+  value: string;
+}
+
+class LeadAutocompleteWrapper extends React.Component<Props, State> {
+  public leadAutocomplete = React.createRef<LeadAutocomplete>();
+  public leadAutocompleteWrapper = React.createRef<HTMLDivElement>();
+
+  public state: State = {
+    isDropdownOpen: false,
+    selectedLead: this.props.lead,
+    value: this.props.lead ? this.props.lead.name : '',
+  };
+
+  public clear = () => {
+    this.setState({
+      value: '',
+    });
+    this.leadAutocomplete.current!.input.current!.focus();
+  }
+
+  public componentDidMount() {
+    this.onSelect(this.state.value, this.props.lead);
+  }
+
+  public render() {
+    const state = this.state;
+    const autocompleteProps = {
+      inputStyle: { ...autocompleteStyles.addLeadInput, width: '450px', marginLeft: '0' },
+      itemsCount: 5,
+      onFocus: this.onAutocompleteFocus,
+    };
+
+    return (
+      <div ref={this.leadAutocompleteWrapper} className={styles.leadAutocomplete}>
+        <ReactSVG className={styles.dealIcon} src={dealIcon} />
+        <LeadAutocomplete
+          {...autocompleteProps}
+          items={this.props.leads}
+          open={state.isDropdownOpen}
+          onBlur={this.onBlur}
+          clear={this.clear}
+          onClear={this.clear}
+          loading={this.props.loading}
+          onChange={this.onChange}
+          onSelect={this.onSelect}
+          value={state.value}
+          ref={this.leadAutocomplete}
+          styles={autocompleteStyles.lead}
+        />
+        {this.props.loading ?
+          <SearchSpinner className={styles.leadSpinner} /> :
+          <ReactSVG
+            className={styles.clearIcon}
+            onClick={this.clear}
+            src={clearIcon}
+          />
+        }
+      </div>
+    );
+  }
+
+  private onSelect = (value: string, item: any) => {
+    this.props.setLead(item._id);
+    this.setState({ value, selectedLead: item, isDropdownOpen: false });
+    this.leadAutocompleteWrapper.current!.removeAttribute('style');
+    this.leadAutocompleteWrapper.current!.blur();
+  }
+
+  private onBlur = () => {
+    this.leadAutocompleteWrapper.current!.removeAttribute('style');
+    this.setState({
+      ...this.state,
+      isDropdownOpen: false,
+      value: this.state.selectedLead.name === this.state.value ? this.state.value : '',
+    });
+    if (this.state.selectedLead.name !== this.state.value) {
+      this.setState({ selectedLead: {} });
+    }
+  }
+
+  private onChange = (event: React.SyntheticEvent) => {
+    const target = event.target as HTMLInputElement;
+    const { value } = target;
+    if (value.length > 1) {
+      this.props.loadSearchResult(value);
+    }
+    this.setState({
+      isDropdownOpen: !isBlank(value),
+      value,
+    });
+  }
+
+  private onAutocompleteFocus = (event: any) => {
+    event.target.parentNode.parentNode.setAttribute('style', 'border: 1px solid #317ae2');
```

Comment:
this would be a simpler solution: `this.leadAutocompleteWrapper.current!.setAttribute('style', 'border: 1px solid #317ae2');`